docs/tutorials/scala/mxnet_java_install_and_run_examples.md

```diff
@@ -0,0 +1,121 @@
+# Install and run Java Examples
+
+## Prerequisites:
+Please follow the Step 1 in the [Scala configuration](http://mxnet.incubator.apache.org/install/scala_setup.html#setup-instructions)
+These should help you install the correct Java version and all dependencies.
+
+## Run the Java example project
+We have provided a general MXNet Java template under `scala-package/mxnet-demo/java-demo` which contains complete instruction on running the Hello world and Object detection examples. Please copy the downloaded MXNet Java package jar file to the `java-demo` folder before build the package there.
```

Comment:
Where do you download the jar file from?

Reply:
For now, the user will need to build from source. We can update this part once we got a SNAPSHOT to publish on Maven